<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>receivingAddress</class>
 <widget class="QDialog" name="receivingAddress">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Receiving Address</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="metier::skins::metier::widget::QrToolButton" name="qrCode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Receiving address:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="addressText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="copyAddress">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/qwidget/assets/Basic-Copy-icon.png</normaloff>:/qwidget/assets/Basic-Copy-icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
 </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>metier::skins::metier::widget::QrToolButton</class>
   <extends>QToolButton</extends>
   <header>metier/skins/metier/widget/qrtoolbutton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
